### ISSUE TITLE
v1.0.0 no longer creates users

### ISSUE THREAD

**Issue by goblain** [okta/okta-sdk-golang#135]
User creation with v1.0.0 no loger works

```go
module ...
go 1.13
require github.com/okta/okta-sdk-golang v1.0.0
```
```go
package main

import (
	"context"
	"fmt"

	okta "github.com/okta/okta-sdk-golang/okta"
)

func main() {
	ctx := context.Background()
	client, _ := okta.NewClient(ctx, okta.WithOrgUrl("..."), okta.WithToken("..."))

	user := &okta.User{
		Profile: &okta.UserProfile{
			"firstName": "goblintest",
			"lastName":  "goblintest",
			"login":     "foo@bar.tld",
			"email":     "foo@bar.tld",
		},
	}
	usr, _, err := client.User.CreateUser(*user, nil)
	if err != nil {
		fmt.Printf("Failed creating OKTA user: %s", err.Error())
	} else {
		fmt.Printf("ID: %s", usr.Id)
	}
```
```bash
$ go run main.go 
Failed creating OKTA user: json: cannot unmarshal object into Go struct field UserCredentials.credentials.emails of type string
```

**Comment by bretterer** [comment]
@goblain thank you for the report. We are currently investigating this issue.

We have released v2.0.0 which has a bunch of new updates, but does include breaking changes, which is why the major version change. Have you had a chance to check out v2?

**Comment by bretterer** [comment]
Closing this as stale.  If you are still experiencing an issue, please feel free to open a new issue and we will get back to you.
